tox.ini

```diff
@@ -49,7 +49,16 @@ passenv =
     AWS_CONTAINER_CREDENTIALS_RELATIVE_URI
     AWS_DEFAULT_REGION
 commands =
-    python -m pytest --ignore=test/integration/sagemaker {posargs}
```

Comment:
We configured it this way so we can any tests with tox. With this change we can't run unit tests or any other tests not under test/integration/local anymore.